build.gradle

```diff
@@ -80,6 +81,29 @@ allprojects {
     }
 }
 
+publishing {
+    publications {
+        pluginZip(MavenPublication) { publication ->
+            pom {
+                name = "opensearch-knn"
+                description = "OpenSearch k-NN plugin"
+                licenses {
+                    license {
+                        name = "The Apache License, Version 2.0"
+                        url = "http://www.apache.org/licenses/LICENSE-2.0.txt"
+                    }
+                }
+                developers {
+                    developer {
+                        name = "OpenSearch"
+                        url = "https://github.com/opensearch-project/opensearch-plugin-template-java"
```

Comment:
url should be url = "https://github.com/opensearch-project/k-NN"

Reply:
ack